project_scrum/project_scrum.py

```diff
@@ -251,7 +256,8 @@ def _resolve_project_id_from_context(self, cr, uid, context = None):
             return context['default_project_id']
         if isinstance(context.get('default_project_id'), basestring):
             project_name = context['default_project_id']
-            project_ids = self.pool.get('project.project').name_search(cr, uid, name=project_name, context=context)
+            project_ids = self.pool.get('project.project').name_search(cr, uid, 
+                                                                       name=project_name, context=context)
```

Comment:
I personally dislike this formatting style. in case you agree with me, you can try breaking the line in a more space efficient way, like this:

```
            project_ids = self.pool.get('project.project').name_search(
                cr, uid, name=project_name, context=context)
```